### PR TITLE
Switch to using real location data

### PR DESCRIPTION
**Pull request by abarth**

### CODE REVIEW

**Comment by abarth** [comment]
@chinmaygarde @jason-simmons

File: examples/hello_services/ios/HelloServices/LocationProvider.m [hunk 1]
```diff
@@ -16,9 +20,16 @@ - (instancetype) init {
...
+        _locationManager = [[CLLocationManager alloc] init];
+        [_locationManager startMonitoringSignificantLocationChanges];
+    }
+
+    CLLocation* location = _locationManager.location;
+
     NSDictionary* response = @{
-        @"latitude": @3.29334,
-        @"longitude": @8.2492492
+        @"latitude": [NSNumber numberWithDouble:location.coordinate.latitude],
```
Comment (chinmaygarde) [inline examples/hello_services/ios/HelloServices/LocationProvider.m]: Nit: `@(location.coodinate.latitude)` would have been fine too.

**Comment by chinmaygarde** [comment]
lgtm

File: examples/hello_services/ios/HelloServices/LocationProvider.m (hunk 1 above)
Comment (abarth) [inline examples/hello_services/ios/HelloServices/LocationProvider.m]: Ah, thanks.

**Pull request merged**